wandb/sdk/launch/agent/agent.py

```diff
@@ -108,6 +109,23 @@ def print_status(self) -> None:
             wandb.termlog(
                 f"{LOG_PREFIX}agent {self._name} polling on project {self._project}, queues {','.join(self._queues)} for jobs"
             )
+        if self._running == 0:
```

Comment:
I think that we could merge the above status prints with this case and the one below. So the agent indicates how many jobs its running, and where its polling in a single output (or at least if statement)

Reply:
Refactored print_status to have less repetition and only a single `termlog` and `_logger.info` call